nesta/packages/crunchbase/predict/.gitignore

```diff
@@ -0,0 +1 @@
+training_data.pickle
```

Comment:
could you move this to the base .gitignore and use `*pickle`, since now we're working with models we don't want to accidentally upload anything we're going to regret...